recipe/meta.yaml

```diff
@@ -59,7 +57,6 @@ extra:
     - xylar
     - jan-janssen
     - jreback
-    - leahecole
```

Comment:
Removing this maintainer, who I believe is no longer active and gets re-invited every time we merge a PR.